src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/AdjournCaseIt.java

```diff
@@ -90,7 +91,7 @@ public void givenCallToMidEventCallbackWithPathYesNoYesThenValidatesDataWithDueD
             List.of(DIRECTIONS_DUE_DATE_PLACEHOLDER),
             List.of(DATE_2019));
 
-        MockHttpServletResponse response = getResponse(getRequestWithAuthHeader(json, CCD_MID_EVENT));
```

Comment:
we still have a midEvent with the "/ccdMidEvent" URL, so unless we're going to remove it, we should also have an assertion on it